src/js/controller/model.js

```diff
@@ -381,6 +388,19 @@ define([
             }
         };
 
+
+        this.setPlaybackRate = function(playbackRate) {
+            if (!_provider || !_.isNumber(playbackRate) || this.get('streamType') === 'LIVE' || !_attached) {
```

Comment:
We shouldn't need a provider to set the rate on the model. The player's rate should be set and providers should be updated when needed.